src/component/calculator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+/* eslint-disable react/prop-types */
+const Buttonfunc = ({ number }) => (
+  <button type="button">{`${number}`}</button>
+);
+
+const ButtonStyle = ({ number }) => (
+  <button type="button" className="style-color">{`${number}`}</button>
+);
```

Comment:
[OPTIONAL] I think you could write this as just one component and pass a prop to style the button. Something like `orange-bg` (bool) and the in the `className` you can do `{ orange-bg ? "style-color" : "" }`